src-frontend/app/components/choice-list.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+    targetObject: Em.computed.alias('parentView'),
```

Comment:
Also in case you're wondering, this line is necessary because components send actions to the controller they reside in by default; not to their parent component if nested (which is what we want).